src/client.js

```diff
@@ -28,7 +28,9 @@ IrcClient.prototype._applyDefaultOptions = function(user_options) {
         nick: 'ircbot',
         username: 'ircbot',
         gecos: 'ircbot',
-        encoding: 'utf8'
+        encoding: 'utf8',
+        ping_interval: 30,
+        ping_timeout: 120,
```

Comment:
I'm doing that to match what the ircd does, but this indeed seems pretty long to me too. This may be useful to survive a DDoS tho, which may be a good idea for bots and bouncers.